general/table.hpp

```diff
@@ -28,166 +28,161 @@ struct Connection
 {
    int from, to;
    Connection() = default;
-   Connection(int from, int to) : from(from), to(to) {}
+   Connection(int from, int to) : from(from), to(to) { }
 
-   bool operator== (const Connection &rhs) const
+   bool operator==(const Connection &rhs) const
    { return (from == rhs.from) && (to == rhs.to); }
-   bool operator< (const Connection &rhs) const
+   bool operator<(const Connection &rhs) const
    { return (from == rhs.from) ? (to < rhs.to) : (from < rhs.from); }
 };
 
 
-/** Data type Table. Table stores the connectivity of elements of TYPE I
-    to elements of TYPE II, for example, it may be Element-To-Face
-    connectivity table, etc. */
+/** @brief Table stores the connectivity of elements of TYPE I to elements of
+    TYPE II. For example, it may be the element-to-face connectivity table. */
 class Table
 {
 protected:
-   /// size is the number of TYPE I elements.
-   int size;
+   int size; ///< The number of TYPE I elements.
 
-   /** Arrays for the connectivity information in the CSR storage.
-       I is of size "size+1", J is of size the number of connections
-       between TYPE I to TYPE II elements (actually stored I[size]). */
-   Memory<int> I, J;
+   /// @name Arrays for the connectivity information in the CSR storage.
+   /// @{
+
+   /// The length of the I array is 'size + 1',
+   Array<int> I;
+
+   /// @brief The length of the J array is equal to the number of connections
+   /// between TYPE I and TYPE II elements.
+   Array<int> J;
+
+   /// @}
 
 public:
    /// Creates an empty table
    Table() { size = -1; }
 
-   /// Copy constructor
-   Table(const Table &);
-
-   /** Merge constructors
-       This is used to combine two or three tables into one table.*/
```

Comment:
Why was this comment deleted?

Reply:
It got moved to the three-table constructor rather than adding a new comment by mistake (previously it was one comment for two constructors). The missing one should be added back.

Reply:
thanks @pazner !